ui/pages/settings/networks-tab/networks-list-item/networks-list-item.js

```diff
@@ -99,7 +101,7 @@ const NetworksListItem = ({
           />
         )
       )}
-      {network.isATestNetwork && (
+      {network.isATestNetwork && network.chainId !== CHAIN_IDS.LINEA_GOERLI && (
```

Comment:
What is the purpose of this condition?

Reply:
Got it, thanks.

It would be great to refactor this to make it more clear that we're trying to render a single icon per network. The way it's setup now is a bit hard to follow. That can wait for another time though

Reply:
Yes I agree :) Thanks